<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>612</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>FTP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(213, 252, 255);</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)；</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FTP地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>输入用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>@ Created by jh、ye11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>输入FTP服务器地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
